cg/server/admin.py

```diff
@@ -640,3 +640,12 @@ class SampleLaneSequencingMetricsView(BaseView):
         "flowcell": FlowcellView.view_flow_cell_link,
         "sample": SampleView.view_sample_link,
     }
+
+
+class OrderView(BaseView):
```

Comment:
I'm not sure we should add a view for it yet since production use them and might be annoyed. But maybe it is worth it for dev purposes? Might be better than dbeaver

Reply:
Oh, I thought we wanted it visible for production. Otherwise I am in favour of removing the view.

Reply:
could be a handy resource also to see the number of cases in a ticket easily. I'd check with production if they have use for something like this already.

Reply:
We don't have any features implemented yet related to an order, so I vote for hiding it for now

Reply:
I think hiding for now is good

Reply:
This does not help with that @ChrOertlin, at least not with the current implementation. The case table already has a ticket_id column. And in the Order table the cases are not visible so the only thing they would gain is that they can go from a case to an order and see customer_id and ordered_at, although an order and a case need not be ordered at the same time I guess.

Reply:
View is hidden now.